package.json

```diff
@@ -1,5 +1,17 @@
 {
+  "name": "opg-lpa",
+  "description": "The Office of the Public Guardian Lasting Power of Attorney online service: Managed by opg-org-infra &amp; Terraform.",
+  "version": "1.0.0",
+  "main": "index.js",
+  "directories": {
+    "doc": "docs",
+    "test": "tests"
+  },
+  "scripts": {
+    "lint:check": "eslint service-front/assets/js cypress/e2e/common"
```

Comment:
Since this reports no further errors with this config, it was a helpful step forward to have the cypress code linted too at this point, as it doesn't interfere with anything else and avoids having to put this back in later and waiting for a further PR. However.... as you wish.... done